src/client/lib/hooks/useAdFreeCookie.ts

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+import { getCookie } from '@guardian/libs';
+
+// User Attributes API cookies are dropped on subscription on support.theguardian.com for use by frontend
+// User Cookie Information: github.com/guardian/members-data-api/blob/main/README.md
+export const useAdFreeCookie = (): boolean => {
+  const [hasAdFree, setHasAdFree] = useState(false);
+  const [isDigitalSubscriber, setIsDigitalSubscriber] = useState(false);
+
+  useEffect(() => {
+    const adfreeCookieExists = !!getCookie({ name: 'GU_AF1' }); // Ad Free session cookie valid 48hrs
```

Comment:
This is a session cookie set by Support. Expiry is in the value which is a Tz but not sure it's worth checking the expiry here.

Reply:
I'm removed the check on the subscriber cookie. This will show base only on the Ad free cookie now. 

https://github.com/guardian/gateway/pull/1589/commits/d54080ceec4a87c2bbd83ae4873cae9ac60451d6